packages/dds/merge-tree/src/index.ts

```diff
@@ -3,21 +3,20 @@
  * Licensed under the MIT License.
  */
 
-export * from "./mergeTree";
-export * from "./ops";
-export * from "./properties";
-export * from "./snapshotlegacy";
-export * from "./collections";
 export * from "./base";
-export { loadSegments } from "./text";
 export * from "./client";
-export * from "./segmentGroupCollection";
+export * from "./collections";
+export * from "./constants";
+export * from "./localReference";
+export * from "./mergeTree";
 export * from "./mergeTreeDeltaCallback";
-export * from "./opBuilder";
 export * from "./mergeTreeTracking";
+export * from "./opBuilder";
+export * from "./ops";
+export * from "./properties";
+export * from "./segmentGroupCollection";
 export * from "./segmentPropertiesManager";
+export * from "./snapshotlegacy";
 export * from "./sortedSegmentSet";
+export { loadSegments } from "./text";
 export * from "./textSegment";
-export * from "./localReference";
-export * from "./snapshotLoader";
```

Comment:
This is mostly an alphabetical sort, but `./snapshotLoader` is no longer exported.